src/llmq/quorums.cpp

```diff
@@ -182,6 +335,66 @@ CQuorumManager::CQuorumManager(CEvoDB& _evoDb, CBLSWorker& _blsWorker, CDKGSessi
     CLLMQUtils::InitQuorumsCache(scanQuorumsCache);
 }
 
+void CQuorumManager::TriggerQuorumDataRecoveryThreads(const CBlockIndex* pIndex) const
+{
+    if (!fMasternodeMode || !CLLMQUtils::QuorumDataRecoveryEnabled() || pIndex == nullptr) {
```

Comment:
argument checks are probably the most expensive (and most likely to be okay) so imo should be last thing to check

```suggestion
    if (!fMasternodeMode || pIndex == nullptr || !CLLMQUtils::QuorumDataRecoveryEnabled()) {
```

Reply:
Hm actually, i guess since `pIndex` can't even be `nullptr` in this case (Because of `UpdateBlockTip` won't ever be called with `nullptr` and i don't *yet* see a reason to trigger it from somewhere else) it should be still better how it is right now, no? That at least was my reason to put them in this order. Basically with the suggested change you would always do a (trivial, but) redundant `pIndex == nullptr` check if  `QuorumDataRecoveryEnabled` is `false` and if it's true it doesn't matter if i don't miss something?